sound/soc/intel/boards/sof_sdw.c

```diff
@@ -916,6 +916,18 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		},
 		.dai_num = 1,
 	},
+	{
+		.part_id = 0x3556,
+		.dais = {
+			{
+				.direction = {true, true},
+				.dai_name = "cs35l56-sdw1",
+				.dai_type = SOF_SDW_DAI_TYPE_AMP,
+				.dailink = {SDW_AMP_OUT_DAI_ID, SDW_AMP_IN_DAI_ID},
```

Comment:
is this intentional that there's no .init  @bardliao ?

If yes, you may want a comment or something in the commit message to that effect.